contracts/ITokenListManager.sol

```diff
@@ -3,27 +3,27 @@ pragma solidity ^0.8.20;
 
 interface ITokenListManager {
 
-    // add token to tokenlist
-    function addToken(address token) external;
+    // Add tokens to tokenlist(whitelist), batch supported
+    function addToken(address[] calldata tokens) external;
```

Comment:
`addTokens` to indicate batch operation.

Please also unify the method name for below several functions.